offer-manager-domain/src/test/groovy/life/qbic/portal/portlet/offers/OfferSpec.groovy

```diff
@@ -708,7 +710,32 @@ class OfferSpec extends Specification {
         affiliation << [externalAffiliation, externalAcademicAffiliation]
     }
 
+    def "Discounts of more than 100% are not possible"() {
+        given: "an item and an offer with this item only"
+        def item = new ProductItem(quantity, ProductFactory.createProduct(productClass, unitPrice, unitPrice))
+        Offer offer = new Offer.Builder(customerWithAllAffiliations, projectManager,  "", "", [item], affiliation).build()
```

Comment:
NIT
```suggestion
        Offer offer = new Offer.Builder(customerWithAllAffiliations, projectManager,  "Offer Title", "The offers description follows here", [item], affiliation).build()
```

Reply:
I think this is not necessary since it should not be taken into account in the rest of the test